tokio/src/task/coop/mod.rs

```diff
@@ -253,11 +253,16 @@ cfg_coop! {
     use std::pin::Pin;
     use std::task::{ready, Context, Poll};
 
+    /// Value returned by the [`poll_proceed`](poll_proceed) method.
+    #[derive(Debug)]
     #[must_use]
-    pub(crate) struct RestoreOnPending(Cell<Budget>);
+    pub struct RestoreOnPending(Cell<Budget>);
```

Comment:
New types should be added to `tests/async_send_sync.rs`.

Reply:
I'm a bit out of my depth here. I added a line for `RestoreOnPending`. Hope I got it right.